docs/best-practices/systables.txt

```diff
@@ -225,7 +225,45 @@ list the hosts and tables with the highest number of rows inside a single
 shard. There's a lot of additional information available for shards. To find
 out more about that, see our :ref:`documentation <import_shards_replicas>`.
 
-Step 5: Managing Snapshots
+
+Step 5: Analyze Allocation Problems
+===================================
+
+Related to the previous step about gaining insights about shards and
+replication is the step about cluster wide shard allocations.
+
+In some circumstances shards allocations behave different than you might expect
+to behave. A typical example is that a table remains underreplicated for some
+reason. In this case you may want to know more about the decision of the
+cluster why certain shards could not be allocated. For that, there is the
```

Comment:
"In this case you may want to know more about the decision of the cluster why certain shards could not be allocated"

this is confusingly worded. can you rephrase?